examples/testapp/src/pages/import-sub-account/components/SendCalls.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Button } from '@chakra-ui/react';
+import { createCoinbaseWalletSDK } from '@coinbase/wallet-sdk';
+import { useCallback, useState } from 'react';
+import { SmartAccount } from 'viem/account-abstraction';
+import { baseSepolia } from 'viem/chains';
+
+export function SendCalls({
+  sdk,
+  subAccount,
+}: {
+  sdk: ReturnType<typeof createCoinbaseWalletSDK>;
+  subAccount: SmartAccount;
+}) {
+  const [state, setState] = useState<string>();
+  const handleSendCalls = useCallback(async () => {
+    if (!sdk) {
+      return;
+    }
+
+    const provider = sdk.getProvider();
+    try {
+      const response = await provider.request({
+        method: 'wallet_sendCalls',
+        params: [
+          {
+            chainId: baseSepolia.id,
+            from: subAccount.address,
+            calls: [],
```

Comment:
what does this actually do?

Reply:
this is for end to end testing for `sendCalls` for imported sub account, it should still work with empty calls, as long as paymaster covers the fee and signing works, we can get a hash back to prove it works